src/Datadog.Trace/ClrProfiler/NativeMethods.cs

```diff
@@ -23,19 +23,37 @@ public static bool IsProfilerAttached()
             return NonWindows.IsProfilerAttached();
         }
 
+        public static void InitializeProfiler(NativeCallTargetDefinition[] methodArrays)
+        {
+            if (IsWindows)
+            {
+                Windows.InitializeProfiler(methodArrays, methodArrays.Length);
+            }
+            else
+            {
+                NonWindows.InitializeProfiler(methodArrays, methodArrays.Length);
+            }
+        }
+
         // the "dll" extension is required on .NET Framework
         // and optional on .NET Core
         private static class Windows
         {
             [DllImport("Datadog.Trace.ClrProfiler.Native.dll")]
             public static extern bool IsProfilerAttached();
+
+            [DllImport("Datadog.Trace.ClrProfiler.Native.dll")]
+            public static extern void InitializeProfiler([In, Out] NativeCallTargetDefinition[] methodArrays, int size);
```

Comment:
Why do we need `[Out]` here? Isn't `[In]` enough? (same comment below)

Reply:
You're right, done in 73f5a81